src/components/searchbox.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import elasticsearch from 'elasticsearch'
+
+let client = new elasticsearch.Client({
+	host: 'localhost:9200',
```

Comment:
The Documentation at [Elastic Search Js API](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/quick-start.html) suggested this way of using the API. and I guess whenever we use elasticsearch server, it uses the same host and port until we change it in the hosts file isn't it ?